### PR TITLE
Fixed small bug (alligned nav items properly according to home page in about and help page)

### PR DESCRIPTION
**Pull request by somesh526**

### CODE REVIEW

**Comment by ashishpapanai** [comment]
@somesh526 Please resolve the conflicts.

**Review by ashishpapanai — APPROVED** [review]

**Comment by somesh526** [comment]
@ashishpapanai  resolved

**Review by ashishpapanai — APPROVED** [review]

File: about.css [hunk 1]
```diff
@@ -12,12 +12,10 @@ body {
...
 .navbar li a{
-    color: #eee;
-    transition: all ease 0.1s;
+   color: #eee;
+   margin-right: 10px;
 }
-
 .navbar li a:hover{
     color: white !important;
     transform: scale(1.2);
```
Comment (ashishpapanai) [inline about.css:21]: As you are working on this fix, Fix this issue as well, the nav items are moving out of the screen when we hover over them on mobile devices.

File: rules.css [hunk 2]
```diff
@@ -11,10 +11,9 @@ body {
...
 .navbar li a{
-    color: #eee;
-    transition: all ease 0.1s;
+ color: #eee;
+ margin-right: 10px;
 }
-
 .navbar li a:hover{
     color: white !important;
     transform: scale(1.2);
```
Comment (ashishpapanai) [inline rules.css:19]: Same fix as done for style.css

File: about.css (hunk 1 above)
Comment (somesh526) [reply about.css:21]: ok sure I haven't checked it

Comment (somesh526) [reply about.css:21]: issue is fixed @ashishpapanai and Thank you

**Review by ashishpapanai — APPROVED** [review]

**Comment by ashishpapanai** [comment]
@somesh526 Create an issue mentioning all the bugs you have fixed with this PR so that I can grade this for SWOC.

**Comment by somesh526** [comment]
do I need to create a separate issue? or I need to mention itself in this PR

**Comment by ashishpapanai** [comment]
> do I need to create a separate issue? or I need to mention itself in this PR

Create a separate issue.

**Pull request merged**